src/manage-apps/fix-packages.md

```diff
@@ -0,0 +1,67 @@
+# Cleaning Up Packages with Terminal Commands
+
+If your system complains about a failed upgrade, package manager conflicts, broken upgrades, or other package-related issues, there are several common fixes to these problems. Some package manager issues can be resolved with the graphical update program, but many require the command line.
+
+## General Fixes for Apt Packages
+
+If a package was installed with `apt`, these commands perform general cleanup that can resolve many of `apt`'s errors. These commands should be run one at a time.
+
+```bash
+sudo apt clean
+sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
+sudo apt full-upgrade
+sudo apt autoremove --purge
+```
+
+- `apt clean` - The `clean` command clears out the local repository of retrieved package files.
+- `apt update -m` - The `-m` option ignores corrupted package files during a package update.
+- `dpkg --configure -a` - The `--configure -a` command configures any unpacked but not yet configured packages.
+- `apt install -f` - attempts to correct a system with broken dependencies
+- `apt full-upgrade` - In addition to fetching package updates, `full-upgrade` downgrades or removes dependencies as necessary when upgrading packages.
+- `apt autoremove --purge` - In addition to removing a package, `autoremove` will remove dependencies no longer required by any installed application. Combining with `purge` will remove any residual configuration files related to the package.
```

Comment:
> In addition to removing a package,

This description doesn't make a lot of sense in this context because you're recommending running it without a package name. I would either change this or add that running it with no package name will remove any packages that were previously dependencies for other installed packages but are no longer required (either because the dependent package was removed, or because the package was updated to no longer depend on certain packages.)